### DISCUSSION TITLE
Mixer pan knobs don't work with plugin.

### DISCUSSION THREAD

**Discussion by bobesponja93** [hydrogen-music/hydrogen#1936] [Hydrogen on Linux]
Trying swh's dj eq plugin. As per the manual, I set the instrument channel strip fader to 0 and the FX knob to the maximum in order to get an all-wet signal. But there's a problem, now panning doesn't work, it's all centered no matter where the pan knob is. It also doesn't matter if I use the stereo version of the plugin. Output is still centered, unpanned.
I also tried with C* Eq10. Same thing, output is centered both with the mono and stereo version of the plugin.
I also tried stereo plugins such as L/C/R delay. Same thing. Output is dead center, no stereo whatsoever.
What's going on?

I'm on Hydrogen 1.0.0-rc1 btw

EDIT: the layer gain knob also doesn't work.

**Comment by trebmuh** [comment]
Hi @bobesponja93 
1.0.0 is quite old now. Have you got the possibility to try with a 1.2.3 which is current?
Please [see here in the "Assets"](https://github.com/hydrogen-music/hydrogen/releases/tag/1.2.3).

**Comment by theGreatWhiteShark** [comment]
Hey @bobesponja93 ,

> What's going on?

The mixer of Hydrogen is quite old and, well, not feature complete. There is a bunch of stuff missing. This includes to properly work with its aux channels. Since implementing various things in order to make it a fully-fledged mixer would be a huge task, it is currently not on our TODO list. But maybe at some point in the future.

I would recommend you to use an external mixer. Personally, I use the one which comes with [MusE](https://github.com/muse-sequencer/muse) and use both applications next to each other when making music.

> EDIT: the layer gain knob also doesn't work.

That might be already fixed in a newer version of Hydrogen. If not, please open a dedicated issue.
